client.go

```diff
@@ -259,11 +274,19 @@ func (s *Client) Run(ctx context.Context,
 		return err
 	}
 
+	// Make sure all required subservers are active.
+	enabled := strings.Join(version.BuildTags, ",")
+	for _, required := range LndRequiredActiveSubservers {
+		if !strings.Contains(enabled, required) {
+			return ErrLndSubserversMissing
```

Comment:
If the user never ran with the right sub-servers, this code may never be hit and a macaroon not found error will terminate the app first. Maybe the version check should be executed before loading all the other macaroons. Perhaps it is a step too far for now.

Reply:
The `versionrpc` uses the default `readonly.macaroon` which should always be there if the node starts correctly.

Reply:
But when we load `readonly.macaroon`, we also try to load all the other macaroons. This happens before we check the tags.

Reply:
Ah, that's what you mean. Yeah, possibly. But I think that does exceed the scope of this PR.

Reply:
But it does make the checking of tags less useful. Especially new users that don't know about build tags will get a macaroon error. I think the whole version/tags check could be integrated nicely in `LndServices` where the `GetInfo` calls is made too (version is info too). At that point, you may be able to read the readonly macaroon outside of the pouch just for this.